src/nile/common.py

```diff
@@ -122,3 +117,18 @@ def is_string(param):
 def is_alias(param):
     """Identiy param as alias (instead of address)."""
     return is_string(param)
+
+
+def get_network_parameter(network):
```

Comment:
I think is a little confusing that a "getter" writes to the env (for code readability). I will suggest changing the name of the function to something like `get_network_param_or_set_env`.

Reply:
I can totally see that. Sounds good!